### DISCUSSION TITLE
TABLE propery SUMMARY

### DISCUSSION THREAD

**Discussion by AcernisSupport** [ant-design-blazor/ant-design-blazor#4056] [General]
Good evening everyone
I am using the TABLE control with the SummaryRow and SummaryCell properties.
Everything works but I can't find the system to:
1 - make sure that the data in the TEXT TYPE (which is now fixed to DANGER and therefore red) can be in another color (e.g. green or blue)
2 - I would like the rows with the SUMMARY properties to have a background color other than black/white but I can't find any part of the code as an example on the site.
Can someone help me?
Thanks everyone and have a good evening

**Comment by AcernisSupport** [comment]
Good morning everyone
I solved it as there is a property that allows you to do it

I close the discussion 

Thank you and good work

**Reply by ElderJames** [reply]
Thank you.
